DESCRIPTION

```diff
@@ -10,7 +10,7 @@ Authors@R:
            family = "Schuster",
            role = c("aut"),
            email = "dschuster@ethz.ch"))
-Description: This package provides useful functions for proteomics data analysis. It can be used for DDA, DIA and SRM data. Both limited proteolysis and regular proteomics experiments can be analysed.
+Description: Useful functions and workflows for proteomics data analysis of both limited proteolysis and regular proteomics experiments. Data generated with search tools such as Spectronaut, MaxQuant and Proteome Discover can be easily used due to flexibility of functions.
```

Comment:
I'd change this to:
"Description: Useful functions and workflows for proteomics quality control and data analysis of both limited proteolysis mass spectrometry (LiP-MS) and regular bottom-up proteomics experiments. Data generated with search tools such as Spectronaut, MaxQuant and Proteome Discover can be easily used due to flexibility of functions."